hazelcast-jet-core/src/test/java/com/hazelcast/jet/impl/util/IOUtilTest.java

```diff
@@ -68,6 +69,9 @@ public void test(Path originalPath) throws IOException {
         Set<Path> originalSet = getChildrenFromRoot(originalPath);
         Set<Path> unzippedSet = getChildrenFromRoot(unzippedPath);
 
+        if (originalSet.size() != unzippedSet.size()) {
```

Comment:
Can we add this to assertCollection method ?

Reply:
you mean to have it on IMDG side or overwrite it on our side ?

Reply:
I thought it comes from JetAssert but still we can improve the one in IMDG to print contents when the size is not equal

Reply:
I've put it to `JetTestSupport` for now, after the change on IMDG side we can remove it